packages/protocol/test-sol/common/GoldToken.t.sol

```diff
@@ -108,7 +108,7 @@ contract GoldTokenTransfer is GoldTokenTest {
   }
```

Comment:
function name needs updating

should read `test_ShouldNotAllowToTransferToNullAddress`

Same for all other similar